src/acquisition/covid_hosp/common/database.py

```diff
@@ -98,33 +87,57 @@ def new_cursor(self):
     finally:
       cursor.close()
 
-  def contains_revision(self, revision):
-    """Return whether the given revision already exists in the database.
+  def update_dataset(self, newer_than=None, older_than=None):
```

Comment:
fixes that result from #1224 may require changes in this method, so we should leave a TODO that points to that PR